packages/@okta/vuepress-site/docs/reference/api/log-streaming/index.md

```diff
@@ -0,0 +1,652 @@
+
+Finds all Log Streams with a [specific type](#log-stream-type)
+
+##### Request example
+
+
+```bash
+curl -v -X GET \
+-H "Accept: application/json" \
+-H "Content-Type: application/json" \
+-H "Authorization: SSWS ${api_token}" \
+"https://${yourOktaDomain}/api/v1/logStreams?filter=type+eq+\"aws_eventbridge\""
+```
+
+##### Response example
+
+```json
+[
+  {
+    "id": "0oa1orzg0CHSgPcjZ0g4",
+    "type": "aws_eventbridge",
+    "name": "Example AWS EventBridge",
+    "lastUpdated": "2021-10-21T16:59:59.000Z",
+    "created": "2021-10-21T16:59:59.000Z",
+    "status": "ACTIVE",
+    "settings": {
+      "accountId": "123456789012",
+      "eventSourceName": "your-event-source-name",
+      "region": "us-east-2"
+    },
+    "_links": {
+      "self": {
+        "href": "https://${yourOktaDomain}/api/v1/logStreams/0oa1orzg0CHSgPcjZ0g4",
+        "method": "GET"
+      },
+      "deactivate": {
+        "href": "https://${yourOktaDomain}/api/v1/logStreams/0oa1orzg0CHSgPcjZ0g4/lifecycle/deactivate",
+        "method": "POST"
+      }
+    }
+  }
+]
+```
+
+
+#### Find Log Streams by type
+
+Finds all Log Streams with a specified status
+
+##### Request example
+
+
+```bash
+curl -v -X GET \
+-H "Accept: application/json" \
+-H "Content-Type: application/json" \
+-H "Authorization: SSWS ${api_token}" \
+"https://${yourOktaDomain}/api/v1/logStreams?filter=status+eq+\"ACTIVE\""
+```
+
+##### Response example
+
+```json
+[
+  {
+    "id": "0oa1orzg0CHSgPcjZ0g4",
+    "type": "aws_eventbridge",
+    "name": "Example AWS EventBridge",
+    "lastUpdated": "2021-10-21T16:59:59.000Z",
+    "created": "2021-10-21T16:59:59.000Z",
+    "status": "ACTIVE",
+    "settings": {
+      "accountId": "123456789012",
+      "eventSourceName": "your-event-source-name",
+      "region": "us-east-2"
+    },
+    "_links": {
+      "self": {
+        "href": "https://${yourOktaDomain}/api/v1/logStreams/0oa1orzg0CHSgPcjZ0g4",
+        "method": "GET"
+      },
+      "deactivate": {
+        "href": "https://${yourOktaDomain}/api/v1/logStreams/0oa1orzg0CHSgPcjZ0g4/lifecycle/deactivate",
+        "method": "POST"
+      }
+    }
+  }
+]
+```
+
+### Update a Log Stream
+
+<ApiLifecycle access="ea" />
+
+<ApiOperation method="put" url="/api/v1/logStreams/${logStreamId}" />
+
+Updates the configuration for a Log Stream
+
+##### Request parameters
+
+| Parameter | Description                       | Param Type | DataType                                      | Required |
+| --------- | --------------------------------- | ---------- | --------------------------------------------- | -------- |
+| logStreamId     | `id` of the log stream to update           | URL        | String                                        | TRUE     |
+| logStream       | Updated configuration for the log stream | Body       | [Log Stream](#log-stream-object) | TRUE     |
+
+All properties must be specified when updating the log stream configuration. Partial updates aren't supported.
```

Comment:
```suggestion
To update a Log Stream configuration, all properties must be specified. Partial configuration updates aren't supported.
```

Reply:
Sorry, actually updatable properties here depend on the type of Log Stream. I'd change it to:

Depending on the type of Log Stream, certain properties may not be updated once the Log Stream is created. Use [Log Stream Schema](/docs/reference/api/schemas/#log-stream-schema-operations) endpoint for the specific Log Stream type to find out which properties are marked as `writeOnce` and cannot be updated.

Reply:
@vanngo-okta ^^